ios/BetterSocial.xcodeproj/project.pbxproj

```diff
@@ -135,13 +135,12 @@
 		88BB34952829E9FB0057417B /* Montserrat-Bold.ttf in Resources */ = {isa = PBXBuildFile; fileRef = 88BB34922829E9DC0057417B /* Montserrat-Bold.ttf */; };
 		88BB6015299BC09D003FA042 /* GoogleService-Info.plist in Resources */ = {isa = PBXBuildFile; fileRef = 88BB6014299BC09D003FA042 /* GoogleService-Info.plist */; };
 		88BB6018299BC0B3003FA042 /* GoogleService-Info.plist in Resources */ = {isa = PBXBuildFile; fileRef = 88BB6016299BC0AF003FA042 /* GoogleService-Info.plist */; };
-		92D8142362E828E0BFD1A9D0 /* BuildFile in Frameworks */ = {isa = PBXBuildFile; };
+		92D8142362E828E0BFD1A9D0 /* (null) in Frameworks */ = {isa = PBXBuildFile; };
 		ADF1D0F61710CB35747750C3 /* libPods-BetterSocial-BetterSocialDev.a in Frameworks */ = {isa = PBXBuildFile; fileRef = 7125237D9815EC140E28774A /* libPods-BetterSocial-BetterSocialDev.a */; };
-		C36230AE8C5C3DD90FCED3EC /* BuildFile in Frameworks */ = {isa = PBXBuildFile; };
+		C36230AE8C5C3DD90FCED3EC /* (null) in Frameworks */ = {isa = PBXBuildFile; };
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

Similar to the previous comment, changes from `BuildFile` to `(null)` in the Frameworks section could potentially cause issues. Ensure that these changes are intentional and won't lead to missing dependencies during build time.

```diff
- 92D8142362E828E0BFD1A9D0 /* BuildFile in Frameworks */ = {isa = PBXBuildFile; };
+ 92D8142362E828E0BFD1A9D0 /* (null) in Frameworks */ = {isa = PBXBuildFile; };

- C36230AE8C5C3DD90FCED3EC /* BuildFile in Frameworks */ = {isa = PBXBuildFile; };
+ C36230AE8C5C3DD90FCED3EC /* (null) in Frameworks */ = {isa = PBXBuildFile; };
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->